stable/memcached/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: memcached
-version: 1.0.0
+version: 2.0.0
```

Comment:
I thought this might count as backwards incompatible functionality by changing defaults, though they probably won't affect anyone and will just be improvements.